Tasks/Task1.js

```diff
@@ -0,0 +1,12 @@
+// function take array as parameter and then retrun a promise that resolve after 3 sec, then print the length of array
+
+const printArrayLength = async (items, callback) => {
+  setTimeout(() => {
+    callback(items.length);
+  }, 3000);
+};
+
+printArrayLength(["A", "B", "C"], (length) => {
+  console.log(length);
```

Comment:
Can you please print a string before print the array lenght for example "Array lenght is : "

Reply:
done.